docs/reference/settings/ml-settings.asciidoc

```diff
@@ -77,40 +78,41 @@ adding an `inference` processor to a pipeline is disallowed. Defaults to `50`.
 
 `xpack.ml.max_machine_memory_percent`::
 (<<cluster-update-settings,Dynamic>>) The maximum percentage of the machine's
-memory that {ml} may use for running analytics processes. (These processes are
-separate to the {es} JVM.) Defaults to `30` percent. The limit is based on the
-total memory of the machine, not current free memory. Jobs are not allocated to
-a node if doing so would cause the estimated memory use of {ml} jobs to exceed
-the limit. When the {operator-feature} is enabled, this setting can be updated
-only by operator users.
+memory that {ml} may use for running analytics processes. These processes are
+separate to the {es} JVM. The limit is based on the total memory of the machine, 
+not current free memory. Jobs are not allocated to a node if doing so would
+cause the estimated memory use of {ml} jobs to exceed the limit. When the
+{operator-feature} is enabled, this setting can be updated only by operator
+users. The minimum value is `5`; the maximum value is `200`. Defaults to `30`.
```

Comment:
We should probably say something about the default being greater than 100 now that we are publicising it.  Something along the lines of this:

> The maximum of `200` is for the special case where it has been determined that using swap space for ML jobs is acceptable.  Do not configure a value higher than the amount of memory left over after running the Elasticsearch JVM unless you have enough swap space configured to accommodate it, and have determined that this is an appropriate configuration for a specialist use case. General best practice is not to use swap on Elasticsearch nodes, so there needs to be a well thought through reason why configuring ML to use it is a good idea in some special situation.

Reply:
Thanks!  I've added this as a tip in https://github.com/elastic/elasticsearch/pull/69947/commits/f9a872480f64391e589c70e765d9ff2bb9cb5041